src/install/installer.ts

```diff
@@ -33,6 +34,17 @@ const removeFolderAsync = util.promisify(removeFolder);
 
 export async function installBrowsersWithProgressBar(packagePath: string) {
   const browsersPath = browserPaths.browsersPath(packagePath);
+  await fsMkdirAsync(browsersPath, { recursive: true });
+  const releaseLock = await lockfile.lock(browsersPath, {
+    retries: {
+      retries: 10,
+      // Retry 20 times during 10 minutes with
+      // exponential back-off.
+      // See documentation at: https://www.npmjs.com/package/retry#retrytimeoutsoptions
+      factor: 1.27579,
```

Comment:
Shouldn't the comment be "Retry 10 times..." and the factor can be simplified to `1.26`?